### PR TITLE
Issue#64

### PR DESCRIPTION
**Pull request by sachaboudjema**

### CODE REVIEW

File: aruba_module_installer/library/modules/network/aruba_clearpass/arubaclearpass_config.py [hunk 1]
```diff
@@ -159,10 +159,10 @@ def main():
     access_token = login_cppm(module, host, client_id, client_secret)
     resp = cppm_api_call(module, host, access_token, api_name, method=method, data=data)
     if resp.code == 200 or resp.code == 201:  # Success
-        if method == "POST":
-            module.exit_json(changed=True, msg=resp.msg, status_code=int(resp.code))
+        if method in ["POST", "PUT", "PATCH"]:
+            module.exit_json(changed=True, msg=resp.msg, status_code=int(resp.code), json=json.load(resp))
         else:
-            module.exit_json(changed=False, msg=result, status_code=int(resp.code))
+            module.exit_json(changed=False, msg=result, status_code=int(resp.code), json=json.load(resp))
```
Comment (karthikeyan-dhandapani) [inline aruba_module_installer/library/modules/network/aruba_clearpass/arubaclearpass_config.py]:
```suggestion
module.exit_json(changed=False, msg=resp.msg, status_code=int(resp.code), json=json.load(resp))
```

Comment (karthikeyan-dhandapani) [reply aruba_module_installer/library/modules/network/aruba_clearpass/arubaclearpass_config.py]: Also result is not a valid variable. Could you please update it to "resp.msg"?

**Comment by sachaboudjema** [comment]
Fixed, this will also resolve #62

**Comment by karthikeyan-dhandapani** [comment]
Also resolves #64

**Pull request merged**